Resources/Prototypes/lobbyscreens.yml

```diff
@@ -9,3 +9,7 @@
 - type: lobbyBackground
   id: Room
   background: /Textures/LobbyScreens/room.png
+  
+- type: lobbyBackground
+  id: Planets
+  background: /Textures/LobbyScreens/planets.png
```

Comment:
I've not messed with lobby screens, just adding this prototype works?

Reply:
It worked on my development build so it probably will work on the release build

Reply:
Yep, that's all it is